### PR TITLE
fix: added test for start with carriage returns

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves https://github.com/ubiquity/ubiquibot-kernel/issues/105
QA: https://github.com/Meniole/command-start-stop/issues/16

### CODE REVIEW

**Comment by gentlementlegen** [comment]
I merge this as it is related to https://github.com/ubiquibot/command-start-stop/pull/44. Let me know if there is any fix needed I'll take care of it.

File: tests/main.test.ts [hunk 1]
```diff
@@ -44,7 +46,20 @@ describe("User start/stop", () => {
 
     const { content } = await userStartStop(context);
 
-    expect(content).toEqual("Task assigned successfully");
+    expect(content).toEqual(SUCCESS_MESSAGE);
+  });
+
+  test("User can start an issue with trimmed command", async () => {
+    const issue = db.issue.findFirst({ where: { id: { equals: 1 } } }) as unknown as Issue;
+    const sender = db.users.findFirst({ where: { id: { equals: 1 } } }) as unknown as PayloadSender;
```
Comment (0x4007) [inline tests/main.test.ts:54]: This uses a database? It should be querying the GitHub api only.

Comment (gentlementlegen) [reply tests/main.test.ts:54]: This is a db used for faking API calls: https://github.com/ubiquibot/command-start-stop/blob/19743ea67faaaea32937791787f000701b8d1cb5/tests/main.test.ts#L3

Comment (0x4007) [reply tests/main.test.ts:54]: That doesn't seem to make sense to me. We should be using a mock http server not a database.

Comment (0x4007) [reply tests/main.test.ts:54]: No idea about all this

**Pull request merged**